ted_sws/data_manager/services/create_notice_collection_materialised_view.py

```diff
@@ -59,6 +59,11 @@ def create_notice_kpi_collection(mongo_client: MongoClient):
     database = mongo_client[config.MONGO_DB_AGGREGATES_DATABASE_NAME or MONGO_DB_AGGREGATES_DATABASE_DEFAULT_NAME]
     notice_events_collection = database[NOTICE_EVENTS_COLLECTION_NAME]
     notice_events_collection.aggregate([
+        {
+            "$match": {
+                "caller_name": "execute",
```

Comment:
I recommand to add $match foreach requiered field, exclude NULL fields with match. 🚀

Reply:
done